<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StarterScreen</class>
 <widget class="QWidget" name="StarterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>935</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_table">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Таблица стран</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_advisor">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>341</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Советы для туристов</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>491</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Справочник для туристов</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
